pages/graph/index.jsx

```diff
@@ -1,10 +1,13 @@
+import BarChart from "@/components/BarChart";
 import DoughnutGraph from "@/components/DoughnutGraph";
 
 function GraphPage() {
   return (
-    <div>
-      <DoughnutGraph />
-    </div>
+    <>
+      <div>
+        <BarChart />
+      </div>
+    </>
```

Comment:
_See my comment above ⬆️_
Since there's only a single component, there's no need for a fragment.

What do you need the `<div>` element for? 🤔
```suggestion
        <BarChart />
```